src/main/java/frc/utils/digitalinput/channeled/ChanneledDigitalInput.java

```diff
@@ -0,0 +1,32 @@
+package frc.utils.digitalinput.channeled;
+
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.utils.digitalinput.DigitalInputInputsAutoLogged;
+import frc.utils.digitalinput.IDigitalInput;
+
+public class ChanneledDigitalInput implements IDigitalInput {
+
+	private final DigitalInput digitalInput;
+	private final Debouncer debouncer;
+	private final boolean inverted;
+
+	public ChanneledDigitalInput(int channel, double debounceTime, boolean inverted) {
```

Comment:
add parameter for the MXP port Digital Inputs.
https://docs.wpilib.org/he/stable/docs/hardware/sensors/serial-buses.html